src/shared/mesh/MeshBuilder.cpp

```diff
@@ -131,6 +131,8 @@ MeshBuilder::operator mfem::Mesh&() { return mesh_; }
 
 MeshBuilder::operator const mfem::Mesh&() const { return mesh_; }
 
+#ifdef TRIBOL_USE_MPI
```

Comment:
Looks like this will enable a serial build, but if a serial build does not use MPI does this break any tests?

Reply:
Serial builds cannot use MPI, so `ParMeshBuilder` should not be exposed to them